### PR TITLE
Landing Page

### PR DESCRIPTION
**Pull request by MarkVed17**
Landing page along with components page which contains the dummy components list.

### CODE REVIEW

File: components-list.html [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+        </div>
+      </a>
+      <a href="" class="component-card">
+        <div class="component-name">
+          <span>Cards</span>
+        </div>
+        <div class="img-container">
+          <img
+            src="/assets/cards.svg"
+            style="width: 144px; height: 124px"
```
Comment (msdeshmukh009) [inline components-list.html:89]: Inline style is really not a great idea. I see you are doing inline styling in lots of places please avoid that.

File: css/components-list.css [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+  justify-content: center;
+  gap: 2rem;
+}
+
+.component-card {
+  display: block;
+  min-width: 400px;
+  max-height: 250px;
+  text-decoration: none;
+  border: 1px solid #35353585;
```
Comment (msdeshmukh009) [inline css/components-list.css]: use color variable maybe?

File: css/index.css [hunk 3]
```diff
@@ -0,0 +1,122 @@
+@import url("https://fonts.googleapis.com/css2?family=PT+Sans+Narrow:wght@400;700&display=swap");
+@font-face {
+  font-family: Jurassic Park;
+  src: url(/assets/JurassicPark-BL48.ttf);
+}
+
+:root {
```
Comment (msdeshmukh009) [inline css/index.css:7]: better to make a single color stylesheet than declare it everywhere.

**Review by msdeshmukh009 — CHANGES_REQUESTED** [review]
Nice theme!! and very unique UI. Great work!!

**Review by vijaytembugade — APPROVED** [review]
Loved the theme. Nice designing. 
Just want to say that , at most of places you used px as units, it would be great if you use rems.

File: components-list.html (hunk 1 above)
Comment (MarkVed17) [reply components-list.html:89]: Yeah, you’re right. I avoid it but not doing it here would rather end up creating multiple class selectors in CSS. I felt doing that inappropriate.

File: css/components-list.css (hunk 2 above)
Comment (MarkVed17) [reply css/components-list.css]: Actually the border on this page isn’t going to be used elsewhere. I had a similar thought.

Comment (msdeshmukh009) [reply css/components-list.css]: But still, it's a better idea to have each and every color in a single stylesheet as a variable. But not a big issue you can decide on your own.

File: components-list.html (hunk 1 above)
Comment (msdeshmukh009) [reply components-list.html:89]: Ohh I see your use case now. Please try to find a way around it whenever you have time.

**Review by msdeshmukh009 — APPROVED** [review]

File: css/components-list.css [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+
+.header-nav-links {
+  margin-left: auto;
+}
+
+.header-nav-links > nav > ul {
+  list-style: none;
+  display: flex;
+  gap: 1rem;
+  padding-right: 10px;
```
Comment (t007rushi) [inline css/components-list.css:66]: use rem for 10 and above px

File: css/components-list.css [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  gap: 2rem;
+}
+
+.component-card {
+  display: block;
+  min-width: 400px;
+  max-height: 250px;
```
Comment (t007rushi) [inline css/components-list.css:100]: same as above

File: css/index.css [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+.header-nav-links {
+  margin-left: auto;
+}
+
+.header-nav-links > nav > ul {
+  list-style: none;
+  display: flex;
+  gap: 1rem;
+  padding-right: 10px;
+  font-size: 2rem;
```
Comment (t007rushi) [inline css/index.css:54]: have consistency for units

File: css/components-list.css [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+  --title-color: #cccccc;
+  --subtitle-color: #808080;
+  --card-bg-color: #27272785;
+  --background-color: #101212;
+}
+
+html,
+body,
+h1,
+p,
```
Comment (t007rushi) [inline css/components-list.css:19]: don't reset tags as it changes the purpose of them

File: css/index.css [hunk 8]
```diff
@@ -0,0 +1,122 @@
...
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+.start-btn:hover {
+  cursor: pointer;
+  background-color: #006f77;
```
Comment (t007rushi) [inline css/index.css:122]: use variables for colors

File: css/index.css [hunk 9]
```diff
@@ -0,0 +1,122 @@
...
+  --primary-color: #00919c;
+  --title-color: #cccccc;
+  --subtitle-color: #808080;
+  --background-color: #101212;
+}
+
+html,
+body,
+h1,
+p,
```
Comment (t007rushi) [inline css/index.css:18]: same here as in comp_list css

**Review by t007rushi — CHANGES_REQUESTED** [review]
I think this would improve code

File: css/components-list.css (hunk 4 above)
Comment (MarkVed17) [reply css/components-list.css:66]: Yeah, I did have a thought on this one. But this was an edge case so I went ahead with px.

File: css/index.css (hunk 6 above)
Comment (MarkVed17) [reply css/index.css:54]: Again. 10 being an edge case, I went ahead with px over rem.

File: css/index.css (hunk 8 above)
Comment (MarkVed17) [reply css/index.css:122]: Isn’t the hover an exception to color variables? I say this as that color won’t be used elsewhere.

File: css/components-list.css (hunk 7 above)
Comment (MarkVed17) [reply css/components-list.css:19]: These reset tags are just restricted to this landing page. Wouldn’t be of much trouble elsewhere.

**Review by tanveertkd — APPROVED** [review]
Love the theme you choose.

File: css/index.css [hunk 10]
```diff
@@ -107,7 +108,7 @@ body {
   letter-spacing: 1px;
   word-spacing: 1px;
```
Comment (tanveertkd) [inline css/index.css:109]: Try using rems wherever you can.

Comment (MarkVed17) [reply css/index.css:109]: Sure thing!👍🏼

**Review by t007rushi — APPROVED** [review]

**Pull request merged**